tests/pr_test.go

```diff
@@ -25,21 +34,92 @@ func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptio
 func TestRunCompleteExample(t *testing.T) {
 	t.Parallel()
 
-	options := setupOptions(t, "mod-template", completeExampleDir)
+	options := setupOptions(t, "mqo", completeExampleDir)
 
 	output, err := options.RunTestConsistency()
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
 
-func TestRunUpgradeExample(t *testing.T) {
+//func TestRunUpgradeExample(t *testing.T) {
+//	t.Parallel()
+//
+//	options := setupOptions(t, "mqo-upg", completeExampleDir)
+//
+//	output, err := options.RunTestUpgrade()
+//	if !options.UpgradeTestSkipped {
+//		assert.Nil(t, err, "This should not have errored")
+//		assert.NotNil(t, output, "Expected some output")
+//	}
+//}
```

Comment:
shouldn't it be enabled? for the first commit let's use the SKIP UPGRADE TEST comment